tslearn/docs/shapelets.rst

```diff
@@ -0,0 +1,82 @@
+.. _shapelets:
+
+Shapelets
+=========
+
+Shapelets are defined in [1]_ as "subsequences that are in some sense
+maximally representative of a class".
+Unformally, if we assume a binary classification setting, a shapelet is said
+discriminant if it is **present** in most series of one class and absent from
+series of the other class.
+To assess the level of presence, one uses shapelet matches:
+
+.. math::
+
+    d(\mathbf{x}, \mathbf{s}) =
+        \min_t \| \mathbf{x}_{t\rightarrow t+L} - \mathbf{s} \|_2
```

Comment:
is this rightarrow notation a standard? I never seen it written like that (I use `:` but I'm probably biased by only writing Python code)

Reply:
I have currently left this as is. I guess it's just a matter of consistency, so if you have used this rightarrow notation everywhere else, then this is definitely the best option :)

Reply:
Yep, it's as you wish. I don't think it's used anywhere else.